packages/kbn-unified-data-table/src/components/custom_control_columns/additional_row_control/row_control_column.tsx

```diff
@@ -29,10 +29,18 @@ export const RowControlCell = ({
 
   const Control: React.FC<RowControlProps> = useMemo(
     () =>
-      ({ 'data-test-subj': dataTestSubj, color, disabled, label, iconType, onClick }) => {
+      ({
+        'data-test-subj': dataTestSubj,
+        color,
+        disabled,
+        iconType,
+        label,
+        onClick,
+        tooltipContent,
+      }) => {
         return (
           <DataTableRowControl size={Size.normal}>
-            <EuiToolTip content={label} delay="long">
+            <EuiToolTip content={tooltipContent} delay="long">
```

Comment:
```suggestion
            <EuiToolTip content={tooltipContent ?? label} delay="long">
```
I wonder if we should force all consumers to pass `tooltipContent`, or just fall back to `label` if not provided? I imagine in most cases they'd be the same.